pubspec.yaml

```diff
@@ -1,15 +1,19 @@
 name: yaml
-version: 2.1.14
-author: "Dart Team <misc@dartlang.org>"
-homepage: https://github.com/dart-lang/yaml
+version: 2.1.15
+
 description: A parser for YAML.
+author: Dart Team <misc@dartlang.org>
+homepage: https://github.com/dart-lang/yaml
+
 environment:
-  sdk: '>=2.0.0-dev.49.0 <2.0.0'
+  sdk: '>=2.0.0-dev.49.0 <3.0.0'
+  
 dependencies:
-  charcode: "^1.1.0"
-  collection: ">=1.1.0 <2.0.0"
-  string_scanner: ">=0.1.4 <2.0.0"
-  source_span: ">=1.0.0 <2.0.0"
+  charcode: ^1.1.0
+  collection: '>=1.1.0 <2.0.0'
+  string_scanner: '>=0.1.4 <2.0.0'
+  source_span: '>=1.0.0 <2.0.0'
+
 dev_dependencies:
-  path: ">=1.2.0 <2.0.0"
-  test: ">=0.12.0 <0.13.0"
+  path: ^1.6.1
+  test: ^1.2.0
```

Comment:
These dependencies also shouldn't be narrowed.

Reply:
Oops, sorry for having missed those. Done